### PR TITLE
Added Fieldset to group generate summary fields.

### PR DESCRIPTION
**Pull request by arshadkhan53**
Added Fieldset to group generate summary fields.

### CODE REVIEW

File: modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module [hunk 1]
```diff
@@ -98,19 +99,28 @@ function ezcontent_smart_article_generate_callback(&$form, FormStateInterface $f
...
+/**
+ * Summary generator callback.
+ *
+ * @todo: Move to a service.
+ */
+function update_summary_field_submit_handler(&$form, FormStateInterface $form_state) {
+  $entity = $form_state->getFormObject()->getEntity();
+  $summary = $form_state->getValue('field_smart_text_summary')[0]['summary_container']['summary_area'];
+  $entity->get('field_smart_text_summary')->summary = $summary;
+  $entity->save();
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module:122]: Do you need to explicitly call Entity->Save ?
Would it not work without it ?

File: modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldWidget/SmartTextareaWithSummaryWidget.php [hunk 2]
```diff
@@ -24,29 +24,46 @@ class SmartTextareaWithSummaryWidget extends TextareaWithSummaryWidget {
    */
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $element = parent::formElement($items, $delta, $element, $form, $form_state);
-    $element['summary_type'] = [
+    $element['summary']['#prefix'] = '<div class="text-summary-wrapper">';
+    $element['summary']['#attributes']['class'] = ['text-summary'];
+    $element['summary']['#attributes']['style'] = 'display: none;';
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldWidget/SmartTextareaWithSummaryWidget.php]: Drupal core provides a class `hidden` to hide things. Please see if you can reuse.
refer - https://git.drupalcode.org/project/drupal/blob/8.8.x/core/modules/system/css/components/hidden.module.css

File: modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module (hunk 1 above)
Comment (arshadkhan53) [reply modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module:122]: I have tested it without Entity->Save  it does not work, we have to save the entity object  after updating it.

**Review by vaibhavjain-in — APPROVED** [review]
Looks good.

**Pull request merged**